### ISSUE TITLE
Badges for contributor milestones

### ISSUE THREAD

**Issue by jessleenyc** [forem/forem#4002]
**Describe the solution you'd like**

Right now we only have one contributor badge awarded when a DEV member makes their contribution. It would be awesome to have two more milestones -- at 32 or 64 PRs merged (we can tweak those numbers).

We're working on badge designs now but anyone is welcome to work on the code part!

**Comment by FeruzOripov** [comment]
Hi, @jessleenyc! How does badges awarding currently works. Can you describe workflow ?

**Comment by Glennmen** [comment]
@FeruzOripov Maybe this can help you in the right direction.

https://github.com/thepracticaldev/dev.to/blob/65fccf00a90647cf27c693ed100d75734e9d0db6/lib/tasks/fetch.rake#L91-L94

https://github.com/thepracticaldev/dev.to/blob/590149fd48d87203ac2e164cc6c3ec6baf957b34/app/labor/badge_rewarder.rb#L63-L75

**Comment by lightalloy** [comment]
Personally, I would even make the number 8 for the badge. (At this point I recalled Hacktoberfest with  4 prs, but it's timebound, so that's another story).

**Comment by JWesorick** [comment]
Some more context here for the next person to work on this issue: plan out how many github api requests you're going to make and make sure it is within the rate limit. I wrote a solution that worked with my spec but then realized it would never work within the rate limit.

**Comment by carlows** [comment]
What's the status of this issue? I see a WIP pull request, is it just blocked by designs? 🤔

**Comment by JWesorick** [comment]
See my comment for some context. I tried implementing it and ran into trouble where I could not figure out how to do the lookups in the github api without running into rate limits. I believe if that gets figured out they would be able to come up with designs for the badges.

**Comment by carlows** [comment]
I saw your comment, but I also saw this open pull request which I was referring to https://github.com/thepracticaldev/dev.to/pull/4007 

I was wondering if it's still being worked on 😄

**Comment by marthaerm** [comment]
Hi! @jessleenyc @rhymes I would like to help with this issue, I noticed that someone started working on it, but it has no recent activity, so I would like to know if it is still open? Thanks!

**Comment by cmgorton** [comment]
Hey @marthaerm sorry for the late reply. This is still open if you are interested in working on it.

**Comment by vishaldeepak** [comment]
I'd like to try this, can I be assigned, thank you

**Comment by rhymes** [comment]
@vishaldeepak, done, thank you!

**Comment by maestromac** [comment]
resolved with https://github.com/forem/forem/pull/14381
